src/components/newsFeed/news.js

```diff
@@ -11,18 +13,21 @@ class News extends React.Component {
     };
   }
 
+  // eslint-disable-next-line class-methods-use-this
+  formatDate(t, timestamp) {
+    const hours = Math.floor(Math.abs(new Date() - new Date(timestamp)) / 36e5);
+    if (hours <= 1) return t('just now');
```

Comment:
```suggestion
    if (hours <= 1) return t('Just now');
```
To be aligned with the capitalized "Yesterday" below